src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -223,8 +224,6 @@ def cli(self,interface="",output=None):
         # Port-channel12 is up, line protocol is up (connected)
         # Vlan1 is administratively down, line protocol is down , Autostate Enabled
         # Dialer1 is up (spoofing), line protocol is up (spoofing)
-        #FastEthernet1 is down, line protocol is down (err-disabled)
```

Comment:
Was this deleted for a reason? It applies to the below regex.

Reply:
This hasn't been deleted in my fork.

https://github.com/ahusking/genieparser/blob/master/src/genie/libs/parser/iosxe/show_interface.py

Reply:
Confirmed code has not been removed. No issue; resolved